security/keycloak/src/main/resources/application.properties

```diff
@@ -3,8 +3,6 @@
 quarkus.oidc.auth-server-url=http://localhost:8180/auth/realms/test-realm
 quarkus.oidc.client-id=test-application-client
 quarkus.oidc.credentials.secret=test-application-client-secret
-# tolerate 1 minute of clock skew between the Keycloak server and the application
```

Comment:
Why are you removing this? I think @Ladicek added it on purpose because of the time sync issue between servers in cluster

Reply:
Because the property `quarkus.oidc.token.expiration-grace` does not exist any longer :/ 
I thought adding the `quarkus.oidc.token.lifespan-grace` property instead, but the tests were consistently working fine.
Any thoughts?

Reply:
Yea I think it was renamed, because it wasn't just for expiration. I'm not sure what's the default grace period, but I'd say that tolerating 1 minute difference is still a good idea (in a test!), even though we've fixed the clock skew issues by configuring correct NTP server.

Reply:
So, do you prefer to get the property back `quarkus.oidc.token.lifespan-grace`?

Reply:
+1 to have quarkus.oidc.token.lifespan-grace, in our env we have time sync issue solved but the TS runs in multiple envs we do not control and it's safer to have this property set to 1 minute

https://quarkus.io/guides/all-config#quarkus-oidc_quarkus.oidc.token.lifespan-grace
quarkus.oidc.token.lifespan-grace: Life span grace period in seconds. When checking token expiry, current time is allowed to be later than token expiration time by at most the configured number of seconds. When checking token issuance, current time is allowed to be sooner than token issue time by at most the configured number of seconds.

Reply:
Agree.

Reply:
Added them back using the new property. Thanks both!